README.md

```diff
@@ -26,6 +26,8 @@
 [k2]: https://riot.im/app/#/room/#ink:matrix.parity.io
 [l1]: https://img.shields.io/discord/722223075629727774?style=flat-square&label=discord
 [l2]: https://discord.com/invite/wGUDt2p
+[s1]: https://img.shields.io/badge/click-white.svg?logo=StackExchange&label=ink!%20Support%20on%20StackExchange&labelColor=white&color=blue
+[s2]: https://substrate.stackexchange.com/questions/tagged/ink/
```

Comment:
This defaults to a filter of unanswered questions. Let's drop people to high quality questions, instead.

```suggestion
[s2]: https://substrate.stackexchange.com/questions/tagged/ink?tab=Votes
```